CONTRIBUTING.md

```diff
@@ -0,0 +1,147 @@
+
+- `author`: Defines the authorship as explained in the [authors section](#authors)
+- `date`: Date when you plan to publish the post, `2006-01-02` format. **Do not forget to update it before publishing it.**
+- `title`: Post title, as it will appear in the very top of the content
+- `image`: Defines the content listing image as explained in the [assets section](#assets)
+- `description`: Short description of the post. Plain text only; no HTML nor markdown allowed.
+- `categories`: An array of at least one of the following: `science`, `technical`, `culture`. If it is not defined, it will not be served by the json blog api, so it will not appear in the source{d} landing.
+- `draft`: This key should not be present in public posts. Read more in the [peer review section](#peer-review)
+
+
+### Formatting the content
+
+It is not allowed to use HTML tags to format the content of the blog.
+
+If you want/need special formatting for your content, please read the [shortcodes tutorial](https://blog.sourced.tech/documentation/shortcodes)
+
+
+### Links to other source{d} blog posts
+
+Links from source{d} blog posts to other source{d} blog posts must be always relative to the blog url. That means that these urls will be like:
+```
+[link text](/post/__POST_FILE_NAME__)
+```
+
+
+### Assets
+
+Given a post content file stored under <br />
+`content/post/__POST_FILE_NAME__.md`,<br />
+its assets should be stored under :<br />
+`static/post/__POST_FILE_NAME__/__ASSET_FILE_NAME_PLUS_EXTENSION__`
+
+That way, the linkable urls would be:<br />
+`/post/__POST_FILE_NAME__/__ASSET_FILE_NAME_PLUS_EXTENSION__`
+
+When it is needed to serve demos with scripts and runnable stuff, the sources must not be hosted in the blog repository. To achieve these usecases, it should be used the [`codepen` shortcode](https://blog.sourced.tech/documentation/shortcodes#codepen)
+
+
+### Authors
+
+Every post entry must define its authorship; to do so, in the content [Front Matter section](#front-matter) it is needed to define the `author` key. 
+
+The `author` key will be one of the authors defined by [the data/authors.yml](data/authors.yml).
+
+The authors entries follows this schema:
+```yaml
+authorKeyName:
+  name: Author Name and Surname
+  thumbnail: https://avatars1.githubusercontent.com/u/__USER_ID__
+  bio: "Short bio/description of the author"
+  social:
+    github: authorUserName
+    twitter: authorTwitterHandler
+```
+
+
+## Peer review
+
+Validate every new contents with [a PR](https://github.com/src-d/blog/pulls), that will be validated by, at least:
+- [vmarkovtsev](//github.com/vmarkovtsev) the content, and
+- [platform](https://github.com/orgs/src-d/teams/platform/members) the layout.
+
+To let your peers to review any new blog post, it is needed to publish it as a **"draft"**. Drafts are only accessible at staging environment http://blog-staging.srcd.run
```

Comment:
s/to review/review/
s/it is needed to publish it/it needs to be published/
s/accessible at/accessible in the/